charts/kargo/templates/api/deployment.yaml

```diff
@@ -118,7 +118,7 @@ spec:
       {{- if or .Values.api.cabundle.configMapName .Values.api.cabundle.secretName }}
       initContainers:
       - name: parse-cabundle
-        image: {{ include "kargo.image" . }}
+        image: alpine:latest
```

Comment:
Probably better to depend on `alpine/curl` latest, which comes with `update-ca-certificates` included. By doing this, we do not have to update and download the package on every Pod start.

Reply:
Resolved in https://github.com/akuity/kargo/pull/2697/commits/2ebcab9c50089ba51871c9e9fb532d17f89c24aa